components/colorloupe/index.css

```
@@ -1,53 +1,97 @@
+  --spectrum-colorloupe-drop-shadow-color: var(--spectrum-color-loupe-drop-shadow-color);
+
+  --spectrum-colorloupe-outer-border-width: var(--spectrum-color-loupe-outer-border-width);
+  --spectrum-colorloupe-inner-border-width: 1px; /* TODO: replace with updated token for color-loupe-inner-border-width */
+  --spectrum-colorloupe-outer-border-color: var(--spectrum-color-loupe-outer-border);
+  --spectrum-colorloupe-inner-border-color: var(--spectrum-color-loupe-inner-border);
 }
 
 .spectrum-ColorLoupe {
-  width: var(--spectrum-colorloupe-width-adjusted);
-  height: var(--spectrum-colorloupe-height-adjusted);
+  inline-size: var(--mod-colorloupe-width, var(--spectrum-colorloupe-width));
+  block-size: var(--mod-colorloupe-height, var(--spectrum-colorloupe-height));
 
   position: absolute;
-  transform: translate(0, var(--spectrum-colorloupe-animation-distance));
+  transform: translate(0, var(--mod-colorloupe-animation-distance, var(--spectrum-colorloupe-animation-distance)));
   opacity: 0;
   transform-origin: bottom center;
-  bottom: calc(50% + var(--spectrum-colorloupe-offset));
-  left: calc(50% - calc(var(--spectrum-colorloupe-width-adjusted) / 2));
+  inset-block-end: calc(var(--spectrum-colorhandle-size) + var(--mod-colorloupe-offset, var(--spectrum-colorloupe-offset)));
+  inset-inline: calc(40% - calc(var(--mod-colorloupe-width, var(--spectrum-colorloupe-width)) / 2));
   transition: transform 100ms ease-in-out, opacity 125ms ease-in-out;
-
   pointer-events: none;
 
+  filter:drop-shadow(
+    var(--mod-colorloupe-drop-shadow-x, var(--spectrum-colorloupe-drop-shadow-x))
+    var(--mod-colorloupe-drop-shadow-y, var(--spectrum-colorloupe-drop-shadow-y))
+    var(--mod-colorloupe-drop-shadow-blur, var(--spectrum-colorloupe-drop-shadow-blur))
+    var(--mod-colorloupe-drop-shadow-color, var(--spectrum-colorloupe-drop-shadow-color))
+  );
+
   &.is-open {
     transform: translate(0, 0);
     opacity: 1;
   }
 }
 
-.spectrum-ColorLoupe-outer {
-  stroke-width: var(--spectrum-colorloupe-outer-border-size);
+.spectrum-ColorLoupe-inner-border {
+  fill: var(--spectrum-picked-color);
+  stroke: var(--mod-colorloupe-inner-border-color, var(--spectrum-colorloupe-inner-border-color));
+  stroke-width: var(--mod-colorloupe-inner-border-width, var(--spectrum-colorloupe-inner-border-width));
 }
 
-.spectrum-ColorLoupe-express {
-  display: var(--spectrum-colorloupe-express-visibility);
+.spectrum-ColorLoupe-outer-border {
+  fill: none;
+  stroke: var(--highcontrast-colorloupe-outer-border-color, var(--mod-colorloupe-outer-border-color, var(--spectrum-colorloupe-outer-border-color)));
+  stroke-width: calc(var(--mod-colorloupe-outer-border-width, var(--spectrum-colorloupe-outer-border-width)) + var(--mod-colorloupe-inner-border-width ,var(--spectrum-colorloupe-inner-border-width)));
 }
 
 .spectrum-ColorLoupe-spectrum {
   display: var(--spectrum-colorloupe-spectrum-visibility);
 }
+
+.spectrum-ColorLoupe-express {
```

```suggestion
.spectrum-ColorLoupe-express {
```

Oh hmm - we've been trying to avoid express-specific naming in our components b/c ideally it will be driven entirely by component tokens being altered.  This is a tricky one.  I'd like to see this moved to the express file in themes somehow. Want to brainstorm on it today?

---

@pfulton Adding you here for your thoughts as well!

---

Yeah, I'm in agreement with @castastrophe - we can/should chat about this, but I'm thinking that SWC would also prefer that we handle situations like this via custom properties.